helper_functions/sunglare_helperFunctions.js

```diff
@@ -0,0 +1,302 @@
+// *---------------*
+// Sunglare querystrings
+// *---------------*
+
+function GetQueryStrings() {
```

Comment:
functions that are not constructors shouldn't be capitalized

Reply:
all functions now start with lower case letter